src/main/java/com/starbucks/controllers/RestController.java

```diff
@@ -1,26 +1,38 @@
 package com.starbucks.controllers;
 
 import com.starbucks.model.Order;
-import com.starbucks.repositary.OrderRepositary;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
+import com.starbucks.service.OrderService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
 
 @org.springframework.web.bind.annotation.RestController
-@RequestMapping("/order")
 public class RestController {
 
-    private final OrderRepositary orderRepositary;
+    private final OrderService orderService;
 
-    public RestController(OrderRepositary orderRepositary) {
-        this.orderRepositary = orderRepositary;
+    public RestController(OrderService orderService) {
+        this.orderService = orderService;
     }
 
-    @PostMapping
+    @PostMapping("/orders")
     public void storeOrder(@RequestBody Order order) {
```

Comment:
Once again, GET, POST as well as PUT should return the response. For POST it should be the created object (somebody will need to show it to the user like an object that User created on Frontend so it's a standard, the same about PUT request)